### PR TITLE
Model ,controller and local database for nsr data

### PR DESCRIPTION
**Pull request by meklitAIA**

### CODE REVIEW

File: app/Filker.php [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Filker extends Model
```
Comment (etsikt) [inline app/Filker.php:7]: Improve readibility: Filker should be Fylker.

Comment (meklitAIA) [reply app/Filker.php:7]: ok

File: app/Http/Controllers/SkolerController.php [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+use App\Skole;
+use Illuminate\Database\Eloquent\Collection;
+use Illuminate\Http\Request;
+
+class SkolerController extends Controller
+{
+    /**
+     * @return Filker[]|Collection
+     */
+    public function all_filke()
```
Comment (etsikt) [inline app/Http/Controllers/SkolerController.php:16]: Should be all_fylke to improve readability.

File: database/migrations/2020_06_10_163549_create_filkers_table.php [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+class CreateFilkersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('filkers', function (Blueprint $table) {
```
Comment (etsikt) [inline database/migrations/2020_06_10_163549_create_filkers_table.php:16]: Should be 'fylkers' to improve readability.

File: database/migrations/2020_06_10_163549_create_filkers_table.php [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::dropIfExists('filkers');
```
Comment (etsikt) [inline database/migrations/2020_06_10_163549_create_filkers_table.php:33]: 'fylkers'

File: routes/api.php [hunk 5]
```diff
@@ -1,6 +1,19 @@
 <?php
 
 
+
+# get schools api
+Route::get('counties', 'SkolerController@all_filke');
```
Comment (etsikt) [inline routes/api.php:6]: all_fylke

Comment (etsikt) [inline routes/api.php:6]: all_fylke

**Pull request merged**